src/lib/spinel/spinel.h

```diff
@@ -1904,6 +1904,25 @@ enum
      */
     SPINEL_PROP_RADIO_COEX_ENABLE = SPINEL_PROP_PHY_EXT__BEGIN + 13,
 
+    /// Calibrated power
+    /** Format: `t(CsU)`
```

Comment:
How about adding documentation that this is a list type property? 
Also can add which commands can be used with it `Insert and `Set(empty) to clear`

Reply:
Done